brainstormforce/ultimate-addons-for-gutenberg
# Fix: CSS not applying on responsive devices and multi-buttons settings not visible in editor issue.

### Description

= 1.25.4 =
* Fix: CSS not applying on responsive devices.
* Fix: Multi Buttons - Child setting are not visible in editor.

### Changes Made
  - Add deviceType attribute in attribute.js
  - Update deviceType attribute withSelect
  - use addBlockEditorDynamicStyles componentDidUpdated
  - Updated css class in styling.js

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Test cases for Responsive CSS
  - Add color or any style to block and check it in tablet/mobile preview.
  - Add a responsive CSS like font size on tablet/mobile and check if it applies in the editor or not.
  - Check style tag is loaded in an iframe when the preview is tablet/mobile.
- Test case for multi-buttons
  - Drag and drop multi-button in master 
  - Click on the button child and check child setting is visible or not.
  - Switch to the above branch and it will get displayed.
- Masonry changes
  - It is already tested and released no need to test.
  
### Checklist:
